content/source/docs/enterprise/before-installing/network-requirements.html.md

```diff
@@ -89,7 +89,13 @@ When [Cost Estimation](/docs/enterprise/admin/integration.html#cost-estimation-i
 
 ## Other Configuration
 
-1. If a firewall is configured on the instance, be sure that traffic can flow out of the `docker0` interface to the instance's primary address. For example, to do this with UFW run: `ufw allow in on docker0`. This rule can be added before the `docker0` interface exists, so it is best to do it now, before the Docker installation.
+1. If a firewall is configured on the instance, be sure that traffic can flow out of the `docker0` interface to the instance's primary address. For example:
+   *  To do this with UFW run: `ufw allow in on docker0`.
+   *  To do this with firewalld run:
+
+        `firewall-cmd --permanent --zone=trusted --change-interface=docker0`
+
+    This rule can be added before the `docker0` interface exists, so it is best to do it now, before the Docker installation.
```

Comment:
I might also actually move this up to be right after the first sentence and before the examples. If we're making a recommendation, we should tell users before they stop reading because they've decided they don't want to do this right now.

```suggestion
    This rule can be added before the `docker0` interface exists, so we recommend adding it before installing Docker.
```